app/controllers/procedures_controller.rb

```diff
@@ -1,58 +1,59 @@
 class ProceduresController < ApplicationController
-     #Currently shows all Procedures
-    def index
-        @procedures = Procedure.all
-    end
+  before_action :authenticate_user!, only: [:edit, :update, :destroy, :base_params, :create_params, :update_params]
+  #Currently shows all pending Procedures
+  def index
+    @procedures = Procedure.where("status = 'approved'")
+  end
     
-    #Creates a new Procedure
-    def new
-        @procedure = Procedure.new
-        @facilities = Facility.all
-    end
+  #Creates a new Procedure
+  def new
+    @procedure = Procedure.new
+  end
 
-    #adds new procedure to database
-    def create
-        procedure = Procedure.new
-        procedure.name = params[:procedure][:name]
-        procedure.date = params[:procedure][:date]
-        procedure.description = params[:procedure][:description]
-        procedure.price = params[:procedure][:price]
-        procedure.facility_id = params[:procedure][:facility_id]
-        procedure.status = "pending"
-        procedure.save
-        redirect_to "/"
-    end
+  #adds new procedure to database
+  def create
+    @procedure = Procedure.create(create_params)
+    @procedure.status = "pending"
+    @procedure.save
+    redirect_to "/"
```

Comment:
use the helper method `root_path`

Reply:
Changed "/" to root_path in admins, items, facilities and procedures controllers.